python/dgl/subgraph.py

```diff
@@ -296,11 +294,12 @@ def _process_edges(etype, e):
         induced_edges.append(_process_edges(cetype, eids))
     sgi = graph._graph.edge_subgraph(induced_edges, preserve_nodes)
     induced_nodes = sgi.induced_nodes
-    return _create_hetero_subgraph(graph, sgi, induced_nodes, induced_edges, store_ids)
+    return _create_hetero_subgraph(graph, sgi, induced_nodes, induced_edges,
+                                   store_nids=store_ids, store_eids=store_ids)
 
 DGLHeteroGraph.edge_subgraph = utils.alias_func(edge_subgraph)
 
-def in_subgraph(g, nodes):
+def in_subgraph(graph, nodes, store_ids=True):
```

Comment:
* Should the default be true or false?
* Should you have two options `store_nids` and `store_eids`?

Reply:
- I set the default to be `True` because storing the original IDs was the default behavior before and this is also consistent with other subgraph extraction APIs like `node_subgraph` and `edge_subgraph`.
- Storing the original node IDs seems to be redundant for `in_subgraph` as a subgraph extracted with this API will have the same set of nodes as the original graph, regardless of whether some nodes will be isolated in the subgraph.

Reply:
> Storing the original node IDs seems to be redundant for in_subgraph as a subgraph extracted with this API will have the same set of nodes as the original graph, regardless of whether some nodes will be isolated in the subgraph. This is consistent with the behavior before.

This behavior is not consistent with `dgl.subgraph`. I think this PR will fix this kind of discrepancy.

Reply:
You mean having the same set of nodes as the original graph?